deploy/crds/mongodb.com_mongodbcommunity_crd.yaml

```diff
@@ -1,31 +1,31 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: mongodb.mongodb.com
+  name: mongodbcommunity.mongodb.com
 spec:
   additionalPrinterColumns:
   - JSONPath: .status.phase
-    description: Current state of the MongoDB deployment
+    description: Current state of the MongoDBCommunity deployment
```

Comment:
I think this can remain as `MongoDB deployment`

Reply:
Done!